internal/service/ec2/ec2_fleet.go

```diff
@@ -70,7 +70,7 @@ func ResourceFleet() *schema.Resource {
 			"excess_capacity_termination_policy": {
 				Type:         schema.TypeString,
 				Optional:     true,
-				Default:      ec2.FleetExcessCapacityTerminationPolicyTermination,
```

Comment:
The EC2 API defines this attribute as optional true with no default value. This might have been set to default by the previous author as a work-around because the API returns a default value for fleet type =`maintain`. However this is not the behaviour when fleet type = `instant`. Handling this check in the read method now. 
<img width="720" alt="Screenshot 2023-01-23 at 19 32 15" src="https://user-images.githubusercontent.com/10087552/214801962-47e2999f-8d73-45c2-99ae-de6e9a1aeeb0.png">